ProjectGenerator/bin/ihp-new

```diff
@@ -234,6 +216,40 @@ if ! [ -x "$(command -v make)" ]; then
 	esac
 fi
 
+if ! [ -x "$(command -v cachix)" ]; then
+	echo -e "\e[1m\e[35mCACHIX MISSING\e[0m"
+	echo -e "IHP uses cachix as a binary cache."
+	echo -e "Learn more about cachix here: \e[4mhttps://cachix.org/\e[0m"
+	echo ""
+	echo ""
+	echo -e "\e[1mWe will install cachix for you now. Continue? (Type y to proceed) \e[0m"
+	read -r -n1 choice
+	echo ""
+	case "$choice" in
+	y | Y) nix-env -iA ${NIXPKGS_ATTR}.cachix ;;
+	*)
+		echo -e "\e[31mPlease install cachix manually and then re-run this program.\e[0m"
+		exit 1
+		;;
+	esac
+fi
+
+if ! grep -s -q "digitallyinduced.cachix.org-1:y+wQvrnxQ+PdEsCt91rmvv39qRCYzEgGQaldK26hCKE=" ~/.config/nix/nix.conf; then
```

Comment:
This is kind of a fragile way to check whether the cache is enabled, but I couldn't figure out anything better.